base/event/FairMCEventHeader.h

```diff
@@ -32,17 +36,22 @@ class FairMCEventHeader : public TNamed
      *@param b        impact parameter [fm] (if relevant)
      *@param nPrim    number of input tracks
      **/
-    FairMCEventHeader(Int_t iEvent, Double_t x, Double_t y, Double_t z, Double_t t, Double_t b, Int_t nPrim);
+    FairMCEventHeader(EventID iEvent, Double_t x, Double_t y, Double_t z, Double_t t, Double_t b, Int_t nPrim);
 
     /** Standard constructor with run identifier **/
     FairMCEventHeader(UInt_t runId);
 
     /** Destructor **/
     virtual ~FairMCEventHeader();
 
+    /// Sentinel value for the event id, meaning no event id stored
+    static constexpr EventID NoEventID = std::numeric_limits<EventID>::max();
+
```

Comment:
Could you move this to the `using EventID`? That way all the EventID related stuff is in one place?